.eslintrc.js

```diff
@@ -22,5 +22,7 @@ module.exports = {
         "newlines-between": "always",
       },
     ],
+
+    "react/jsx-no-leaked-render": [ERROR, { validStrategies: ["ternary"] }],
```

Comment:
Some folks may disagree putting this in a linting rule. But I'm good with it 👍

Reply:
I'm one of them :/

There are a few occurrences where the evaluated value is clearly a boolean (search.hasParam()), and I don't see a reason to be **forced** to write a ternary for it..

Reply:
My vision about this is: although it's not necessary when it's a boolean condition, it's inconsistent if we allow both.
You could argue that it's not a problem per se, but this allows people to make mistakes against this rule & thus create bugs.

Not having to think about what to do (as you always use ternaries), gives a lower cognitive load + makes sure you'll _never_ have bugs.

Reply:
I'd hear you for a specific app, but thee are examples .. anyways 🤷🏼‍♂️

Reply:
I think it's important to make good suggestions to people looking at our examples/templates

Reply:
@MichaelDeBoey Let's remove the lint rule to merge this. In general I think we're trying to move away from opinionated lint rules in examples and templates. Not arguing the value of this in particular, just easier to avoid the distracting conversations. Eventually I'd like to update all examples to use our eslint package and nothing else.